src/mpl2/src/object.h

```diff
@@ -778,17 +778,18 @@ struct Rect
     f_y = f_y_;
   }
 
-  bool isValid() const
+  bool isValid() const { return (lx < ux) && (ly < uy); }
+
+  void mergeInit()
   {
-    return (lx > 0.0) && (ly > 0.0) && (ux > 0.0) && (uy > 0.0);
+    lx = std::numeric_limits<float>::max();
+    ly = lx;
+    ux = std::numeric_limits<float>::min();
```

Comment:
This is a classic C++ trap.  You want ::lowest not ::min.  More annoyingly ::min works fine for integral types.